src/Application/Common/Models/Dtos/UserDto.cs

```diff
@@ -0,0 +1,24 @@
+using Application.Common.Mappings;
+using AutoMapper;
+using Domain.Entities;
+
+namespace Application.Users.Queries;
+
+public class UserDto : IMapFrom<User>
+{
+    public string Username { get; set; }
+    public string Email { get; set; }
+    public string FirstName { get; set; }
+    public string LastName { get; set; }
+    public Guid DepartmentId { get; set; }
+    public string Role { get; set; }
+    public string Position { get; set; }
+    public bool IsActive { get; set; }
```

Comment:
this should be `IsActivated`!

Reply:
IsActive is different from IsActivated according to @ChienNQuang

Reply:
@ChienNQuang can you explain?